addons/website_event/controllers/main.py

```diff
@@ -274,25 +286,28 @@ def registration_tickets(self, event, slot_id):
             }
         })
 
-    @http.route(['/event/<model("event.event"):event>/registration/new'], type='jsonrpc', auth="public", methods=['POST'], website=True)
-    def registration_new(self, event, **post):
+    @http.route(['/event/<model("event.event"):event>/registration/tickets/confirm'], type='http', auth="public", methods=['POST'], website=True)
+    def registration_tickets_confirm(self, event, **post):
```

Comment:
Maybe include a mapping of old (if any) -> new equivalent route in your commit message. (Could we use the old name, if appropriate, to reduce change? :thinking: )